tests/shared/utils.py

```diff
@@ -96,7 +96,7 @@ def ping(self, node_id: str, port: int = 8081, timeout: int | None = 5):
         LOGGER.debug(f"GET Ping node {node_id} ({self.container.name} - {self.node_id})")
         port = self.__external_port(port)
         response: requests.Response = requests.get(
-            f"http://localhost:{port}/ping/{node_id}", headers=http_client_headers, timeout=timeout
+            f"http://localhost:{port}/ping/reliable/{node_id}", headers=http_client_headers, timeout=timeout
```

Comment:
Maybe add arg `transport: str = "reliable"`
and then `f"http://localhost:{port}/ping/{transport}/{node_id}"`
?